pkg/machines/components/vmSnapshotsTab.jsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from "moment";
+
+import cockpit from 'cockpit';
+import { vmId } from "../helpers.js";
+import { CreateSnapshotModal } from "./vmSnapshotsCreateModal.jsx";
+import { ListingTable } from "cockpit-components-table.jsx";
+import { Button, Tooltip } from '@patternfly/react-core';
+import { InfoAltIcon } from '@patternfly/react-icons';
+
+import './vmSnapshotsTab.css';
+
+const _ = cockpit.gettext;
+
+function prettyTime(unixTime) {
+    moment.locale(cockpit.language, {
+        longDateFormat : {
+            LT: "HH:mm:ss",
```

Comment:
(I don't know if this actually gets set here, but as a non-programmer, I'm kind of shit at reading code, I'll put this comment here :)

I don't think we need the seconds here.
I also have all other time/date occurrences on this test VM formatted as AM/PM, but the snapshots seems to stick to 24-hour clock.

Reply:
Fixed this and all above mentioned